### DISCUSSION TITLE
如何自动配置符文/召唤师技能呢?

### DISCUSSION THREAD

**Discussion by sugarnine9** [cangzhang/ChampR#157] [General]
最近看到一个app《游戏魔方》，能够自动设置符文，自动配置召唤师技能（预先设置闪现键在哪个）。这个功能最先看到好像是在opgg的客户端上，能够自动配置符文和召唤师技能。这个功能能够在最后几秒换英雄，或者忘记点符文、换召唤师技能的情况下非常好用，不知道开发者有没有这样的想法呢？

新年快乐！

**Comment by DuvyDev** [comment]
That would be great!! I hope they implement it soon!
